exotica_core/src/kinematic_tree.cpp

```diff
@@ -1007,8 +1066,32 @@ void KinematicTree::SetFloatingBaseLimitsPosXYZEulerZYX(
         ThrowPretty("Wrong limit data size!");
     }
     for (int i = 0; i < 6; ++i)
+    {
+        // Set velocity and acceleration to +inf
+        controlled_joints_[i].lock()->joint_limits = {lower[i], upper[i]};
+        controlled_joints_[i].lock()->velocity_limit = {inf};
+        controlled_joints_[i].lock()->acceleration_limit = {inf};
```

Comment:
Check ```velocity.size()``` and ```acceleration.size()```.
If zero, assign infinity, if !=6 throw error.

Reply:
This doesn't need to check the size of velocity or acceleration as they are not arguments of the function? This is the 2 arg version of the overloaded function for if velocity and acceleration are not provided - it just sets them to inf. You told me to keep this so that it doesn't break the old API.